package-system/libpng/build_config.json

```diff
@@ -1,67 +1,113 @@
 {
     "git_url": "https://git.code.sf.net/p/libpng/code",
     "git_tag": "v1.6.37",
-    "package_name": "libpng",
-    "package_version": "1.6.37-rev1",
+    "package_name": "png",
```

Comment:
Why the change in package name from `libpng` to `png`?

Reply:
"png" is the actual official package name from CMake ( https://cmake.org/cmake/help/latest/module/FindPNG.html ) so that find_package(PNG) works.  I wanted to standardize in such that find_package(x) is serviced by package x-version-rev-platform to simplify things.

I could have made it "PNG" but most of our packages (if not all?) are lowercase to avoid confusion.    Note that the cmake target is still PNG and I will be updating o3de side to change from 3rdParty::libpng to 3rdParty::PNG to conform to cmake's built-in findPng, so it can be a drop in replacement for other 3p libs like OpenImageIO